server/src/web_rtc_signaling_server.ts

```diff
@@ -102,12 +102,28 @@ export class WebRTCSignalingServer {
   ): Promise<void> {
     await this.connectionStore.addConnection(log, walletAddress, connection);
     log.info(`addUserConnection walletAddress: ${walletAddress}}`);
+    await sendAnalytics(
```

Comment:
await Promise.all([]) these two promises so they can run in parallel

Reply:
fixed